src/utils/wallet.js

```diff
@@ -211,8 +211,15 @@ class Wallet {
                 balance: await this.getBalance(),
                 accountId: this.accountId,
                 accounts: this.accounts,
-                authorizedApps: accessKeys.filter(it => it.access_key && it.access_key.permission.FunctionCall),
-                fullAccessKeys: accessKeys.filter(it => it.access_key && it.access_key.permission === 'FullAccess'),
+                authorizedApps: accessKeys.filter(it => (
+                    it.access_key 
+                    && it.access_key.permission.FunctionCall 
+                    && it.access_key.permission.FunctionCall.receiver_id !== this.accountId
```

Comment:
@marcinbodnar wonder if we should start using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining given we use Babel anyway

Reply:
Yeah, sounds good.